sphinx/util/osutil.py

```diff
@@ -205,14 +205,21 @@ def ustrftime(format, *args):
             return r.encode().decode('unicode-escape')
 
 
-def safe_relpath(path, start=None):
+def relpath(path, start=os.curdir):
```

Comment:
I fixed the default value of `start` argument. It should be same as `os.path.relpath()`: https://docs.python.org/3.6/library/os.path.html#os.path.relpath